### ISSUE TITLE
KDE Plasma tray icon blank square

### ISSUE THREAD

**Issue by bundito** [patrikx3/onenote#34]
It worked at first, but now the KDE tray icon has regressed into a plain white square. I've tried it with both light and dark themes, but no luck. I can still click it and get the dropdown of functions.

File it under "medium-low"...

![screenshot_20180914_104605](https://user-images.githubusercontent.com/1828240/45560687-701e8700-b80b-11e8-8eee-37cc902ec6bc.png)

**Comment by bundito** [comment]
PNGs are supposed to be fine. What KDE really loves are SVG icons. But I'm not asking you to do that.

My distro is a developer's git-unstable installation, so it's more likely something on "our" end changed. 

Thanks for investigating.

**Comment by bundito** [comment]
I didn't mean any disrespect. I tried them both and it was the same result, so it's likely an issue with KDE. Things are changing daily.

I closed it because I didn't want you to spend any more time on it. It worked a few days ago; as the KDE PIasma devs keep working, it'll probably start working again like magic. didn't realize those builds were just for me. Thank you, that was very generous. 

Have a good weekend and thank you for your efforts.
